SU2_CFD/include/solvers/CFVMFlowSolverBase.hpp

```diff
@@ -137,37 +137,37 @@ class CFVMFlowSolverBase : public CSolver {
   su2double Total_Heat = 0.0;           /*!< \brief Total heat load for all the boundaries. */
   su2double Total_MaxHeat = 0.0;        /*!< \brief Maximum heat flux on all boundaries. */
   su2double AllBound_CNearFieldOF_Inv = 0.0; /*!< \brief Near-Field press coeff (inviscid) for all the boundaries. */
-  su2double* CNearFieldOF_Inv = nullptr;     /*!< \brief Near field pressure (inviscid) for each boundary. */
-  su2double* Surface_HF_Visc = nullptr;      /*!< \brief Total (integrated) heat flux for each monitored surface. */
-  su2double* Surface_MaxHF_Visc = nullptr;   /*!< \brief Maximum heat flux for each monitored surface. */
-  su2double* HF_Visc = nullptr;              /*!< \brief Heat load (viscous contribution) for each boundary. */
-  su2double* MaxHF_Visc = nullptr;           /*!< \brief Maximum heat flux (viscous contribution) for each boundary. */
-  su2double AllBound_HF_Visc = 0.0;          /*!< \brief Heat load (viscous contribution) for all the boundaries. */
-  su2double AllBound_MaxHF_Visc = 0.0;       /*!< \brief Maximum heat flux (viscous contribution) for all boundaries. */
-
-  su2double** Inlet_Ptotal = nullptr;      /*!< \brief Value of the Total P. */
-  su2double** Inlet_Ttotal = nullptr;      /*!< \brief Value of the Total T. */
-  su2double*** Inlet_FlowDir = nullptr;    /*!< \brief Value of the Flow Direction. */
-  su2double** HeatFlux = nullptr;          /*!< \brief Heat transfer coefficient for each boundary and vertex. */
-  su2double** HeatFluxTarget = nullptr;    /*!< \brief Heat transfer coefficient for each boundary and vertex. */
-  su2double*** CharacPrimVar = nullptr;    /*!< \brief Value of the characteristic variables at each boundary. */
-  su2double*** CSkinFriction = nullptr;    /*!< \brief Skin friction coefficient for each boundary and vertex. */
-  su2double** WallShearStress = nullptr;   /*!< \brief Wall Shear Stress for each boundary and vertex. */
-  su2double*** HeatConjugateVar = nullptr; /*!< \brief CHT variables for each boundary and vertex. */
-  su2double** CPressure = nullptr;         /*!< \brief Pressure coefficient for each boundary and vertex. */
-  su2double** CPressureTarget = nullptr;   /*!< \brief Target Pressure coefficient for each boundary and vertex. */
-  su2double** YPlus = nullptr;             /*!< \brief Yplus for each boundary and vertex. */
-
-  bool space_centered;       /*!< \brief True if space centered scheeme used. */
+  vector<su2double> CNearFieldOF_Inv;    /*!< \brief Near field pressure (inviscid) for each boundary. */
```

Comment:
Hi!,  I noticed that these codes from line  140 assign every boundary marker a container, even through these boundaries don't need these variable. I'm not sure if I'm understanding this correctly。